doc/quick-guide.md

```diff
@@ -791,19 +791,126 @@ These are not infix operators, but look like function calls, as in
 ## Bit-wise Operators on Strings
 
 The shift and bitwise operators also apply to strings, where they perform
-byte-wise operations.
-
-Shifting strings to the left has the effect to fill zero bytes in the right,
-and shifting strings to the right has the effect to remove characters from
-the right. Logical shift operators also fill zero bytes in the other side, so
-the lengths of their operands are unchanged.
-
-Byte-wise AND, OR and XOR operations are intuitively defined on two strings
-of the same length. If one string is longer than the other, the longer part
-corresponds to the 'missing information' of the other string. The byte-wise
-AND operator trims the longer string, and produces a result of the same
-length as the shorter one; while the byte-wise OR and XOR operators treat
-'missing information' as zeroes (which means to copy from the longer string),
-and produce a result of the same length as the longer one.
+byte-wise operations:
+
+``` node
+#1:1> "12345678"
+* running 'expression #1'...
+* result #1: string(8) "12345678";
+```
+
+Shifting strings to the left (`<<`) has the effect to fill zero bytes in the
+right;
+Shifting strings to the right (`>>`) has the effect to remove characters from
```

Comment:
If the line above ends with a semicolon, then the next sentence shall start with a lowercase letter, and shall not start on its own line.

The parentheses are misleading; they look as if `<<` and `>>` are the only shift operators. These sentences apply to all shift operators.